pom.xml

```diff
@@ -187,37 +185,14 @@
     <!-- Do not add compile dependencies here. This can cause problems for libraries that depend on
          io.zipkin.zipkin2:zipkin difficult to unravel. -->
 
-    <dependency>
-      <groupId>junit</groupId>
```

Comment:
Skimmed through a few files for kicks but this is probably the main change and if build is green then it looks like it worked well